### ISSUE TITLE
[Supporting the platform - Server request>Inspect]: Controls present in "Object" section under "Query" tab are not visible after resizing the page to 200%.

### ISSUE THREAD

**Issue by natellium** [grafana/grafana#66584]
## User Experience: 
Users who have vision issues will be affected if controls are not visible or truncated after increasing the screen zoom to 200%. They will not be able to access the controls present on the screen and might get disoriented. 

## Pre-requisite:  
resize the page to 200%

## Repro Steps:
Open #URL: https://play.grafana.org/. Sign in with valid credentials. "Home" page appears. Navigate the "Server Requests Graphs". Expand the button and select the "Inspect" button. Select "Data" menu item. Navigate all the controls under the "Query" tab pane. Select the "Refresh" button. Select "+Expand all" button. Navigate to "Object " section. Verify the issue.

## Actual Result: 
Controls present in "Object" section under "Query" tab are not visible after resizing the page to 200%.
Observation: Controls such as "request: object" and "response: Array [4]" and contents inside them present under "Object" section are not visible when the page resizes to 200%.

## Expected Result:
 All the controls present in "Object" section under "Query" tab should be clearly visible after resizing the page to 200%.

## Severity
`SEV2`

## Area
`TestRuleResult Component (Alerting Inspect Drawer) `

**Comment by gillesdemey** [comment]
I think this one has been incorrectly labelled; after trying to repro based on the steps above there's no involvement from any Alerting components :)

**Comment by natellium** [comment]
You're totally right @gillesdemey, thanks for pointing it out :)
